TheThingsNetwork/lorawan-stack
# Allow admins to see all gateways and applications in the Console

#### Summary


Closes https://github.com/TheThingsNetwork/lorawan-stack/issues/2444

<details>
<summary>Screenshots</summary>

non-admin user:

<img width="1176" alt="Screenshot 2020-06-21 at 18 21 59" src="https://user-images.githubusercontent.com/16374166/85228516-78e2ba80-b3ec-11ea-8051-6eb569133bb3.png">

.

<img width="1176" alt="Screenshot 2020-06-21 at 18 23 42" src="https://user-images.githubusercontent.com/16374166/85228521-7bddab00-b3ec-11ea-81ce-cd9556b04a10.png">


admin user:

<img width="1176" alt="Screenshot 2020-06-21 at 18 22 24" src="https://user-images.githubusercontent.com/16374166/85228517-7a13e780-b3ec-11ea-96d3-51f77269d360.png">

.

<img width="1176" alt="Screenshot 2020-06-21 at 18 22 35" src="https://user-images.githubusercontent.com/16374166/85228519-7b451480-b3ec-11ea-93c7-1150f42bbffc.png">

.

<img width="1176" alt="Screenshot 2020-06-21 at 18 24 13" src="https://user-images.githubusercontent.com/16374166/85228522-7c764180-b3ec-11ea-971e-ecfaf9a791d4.png">

.

<img width="1176" alt="Screenshot 2020-06-21 at 18 24 21" src="https://user-images.githubusercontent.com/16374166/85228523-7c764180-b3ec-11ea-997b-6eff91cbbd0e.png">



</details>

#### Changes


- Add some styling tweaks to the `<FetchTable />` component
- Adjust `<ApplicationsTable />` and `<GatewaysTable />`


#### Testing



1. Check app/gtw table behavior for non-admin user
2. Check app/gtw table behavior for admin user

##### Regressions



`<FetchTable />` styling and general behavior



#### Checklist
